common/common.mk

```diff
@@ -19,17 +19,8 @@ RUNC_REMOTE       ?=https://github.com/opencontainers/runc.git
 REF?=HEAD
 
 # Select the default version of Golang and runc based on the containerd source.
-# The runc version commit/tag is defined in vendor.conf. Code below is based on
-# the code used in the containerd repository:
-# https://github.com/containerd/containerd/blob/499fbb0337c9138b5360117e0b25a7a1428f9667/script/setup/install-runc#L24
-ifdef CONTAINERD_DIR
-GOVERSION?=$(shell grep "ARG GOLANG_VERSION" $(CONTAINERD_DIR)/contrib/Dockerfile.test | awk -F'=' '{print $$2}')
-RUNC_REF?=$(shell grep "opencontainers/runc" "$(CONTAINERD_DIR)/vendor.conf" | awk '{print $$2}')
-else
-# TODO adjust GOVERSION and RUNC_REF macro to take CONTAINERD_REMOTE into account
-GOVERSION?=$(shell curl -fsSL "https://raw.githubusercontent.com/containerd/containerd/$(REF)/contrib/Dockerfile.test" | grep "ARG GOLANG_VERSION" | awk -F'=' '{print $$2}')
-RUNC_REF?=$(shell curl -fsSL "https://raw.githubusercontent.com/containerd/containerd/$(REF)/vendor.conf" | grep "opencontainers/runc" | awk '{print $$2}')
-endif
+GOVERSION?=$(shell grep "ARG GOLANG_VERSION" src/github.com/containerd/containerd/contrib/Dockerfile.test | awk -F'=' '{print $$2}')
```

Comment:
After checked the release/1.4.4, the `contrib/Dockerfile.test` isn't updated to go1.15.8. Is it possible to use `.github/workflows/release.yml`?

Reply:
Hmm.. I think we should fix that; if the Dockerfile.test is out of sync, that's a mistake

Reply:
Actually; it looks up-to-date; https://github.com/containerd/containerd/blob/v1.4.4/contrib/Dockerfile.test#L9

```dockerfile
ARG GOLANG_VERSION=1.15.8
```

That said; I think this should be treated as a default (to make things work "out of the box"), but probably the recommendation should be to pass these variables when building 🤔

Reply:
Sorry my mistake. I used the makefile to build and the script uses 1.15.5 goversion...